test/models/CommentModel/test.js

```diff
@@ -0,0 +1,84 @@
+const {
+  expect, CommentModel, faker, sinon, Mongoose,
+} = require('test/testHelper');
+const { CommentFactory } = require('test/factories');
+const _ = require('lodash');
+
+describe('Comment Model', async () => {
+  describe('On getOne', async () => {
+    let comment, commentCount, nameAuthor;
+    beforeEach(async () => {
+      commentCount = 6;
+      nameAuthor = faker.name.firstName();
+      for (let iteration = 0; iteration < commentCount; iteration++) {
+        if (iteration === 0) comment = await CommentFactory.Create({ author: nameAuthor });
+        else await CommentFactory.Create({});
+      }
+    });
+    it('Should check comment for identity using author key', async () => {
+      const result = await CommentModel.getOne({ ..._.pick(comment, 'author', 'permlink') });
+      expect(result.comment).to.deep.eq(comment);
+    });
+    it('Should check comment for identity using userId key ', async () => {
+      const result = await CommentModel.getOne({ ..._.pick(comment, 'permlink', 'userId') });
+      expect(result.comment).to.deep.eq(comment);
```

Comment:
to BE deep eq! Why do you turn ({ ..._.pick(comment, 'author', 'permlink') }) into an object, and then spread it?!?! This is not a necessary operation, _.pick(comment, 'author', 'permlink')

Reply:
const result = Use destructurization!!! const {comment} = (if you have conflict of names, you can write const {comment : receivedComment} = )